v2/internal/controllers/crd_cache_redis_20230801_test.go

```diff
@@ -0,0 +1,283 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	cache "github.com/Azure/azure-service-operator/v2/api/cache/v1api20230801"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1api20200601"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_Cache_Redis_20230801_CRUD(t *testing.T) {
```

Comment:
Since we're adding this test, wondering if we should remove the test and recordings for `Test_Cache_Redis_20201201_CRUD`, since it's old and IIRC we decided to only keep around ~2 GA versions and ~1 preview version tests?

Reply:
Done!